testworkflow.go

```diff
@@ -701,6 +705,7 @@ func NewTestWorkflow(client daisycompute.Client, computeEndpointOverride, name,
 	t.Client = client
 
 	var err error
+	t.testExcludeFilter = flag.Lookup("exclude_discrete_tests").Value.(flag.Getter).Get().(string)
```

Comment:
Can you add this to the function signature so that the library does not have a dependency on the manager? Otherwise LGTM and ready to merger

Reply:
Sorry @a-crate, I got a bit sidetracked :smile: Is this what you meant?

Reply:
Yep, thanks